android-ktx/build.gradle

```diff
@@ -36,17 +36,17 @@ configurations.all {
 dependencies {
     api "com.mirego.trikot:trikotFoundation:$trikot_foundation_version"
     api "com.mirego.trikot:streams:$trikot_streams_version"
-    api 'com.mirego.trikot:http:0.24.1'
+    api 'com.mirego.trikot:http:1.0.0-SNAPSHOT'
```

Comment:
Maybe not the snapshot :)

Reply:
For the moment every dependency is pointing to a snapshot to test it integrated in a project. I will update the references before merging.

Reply:
Finally I decided to publish the non snapshot version to my maven local to avoid forgetting a reference somewhere and updated those values.